pages/agent/v3/tracing.md.erb

```diff
@@ -0,0 +1,24 @@
+# Tracing in the Buildkite Agent
+
+Distributed tracing tools like [Datadog APM](https://www.datadoghq.com/product/apm/) or [OpenTelemetry](https://opentelemetry.io/) tracing allow you to gain more insight into the the performance of your CI runs - what's fast, what's slow, what could be optimized, and more importantly, how these things are changing over time.
+
+The Buildkite agent currently supports the two tracing backends listed above, Datadog APM (via OpenTracing) and OpenTelemetry. This doc will guide you through setting up tracing using either of these backends.
+
+{:toc}
+
+## Using Datadog APM
+
+To use the Datadog APM integration, start the Buildkite agent with the `--tracing-backend datadog` option. This will enable Datadog APM tracing, and send the traces to a Datadog agent at `localhost:8126` by default. If your Datadog agent is located at another host, the Buildkite agent will respect the `DD_AGENT_HOST` and `DD_AGENT_TRACE_PORT` environment variables defined by [`dd-trace-go`](https://docs.datadoghq.com/tracing/setup_overview/setup/go/?tab=containers). Note that there will need to be a Datadog agent present at the above address to ingest these traces.
+
+Once this is done, the agent will start sending tracing information to Datadog. You can observe traces as they come in in the APM > Traces screen in your Datadog instance.
+
+## Using OpenTelemetry Tracing
+
+<div class="Docs__wip-note">
+  <p class="Docs__wip-heading">Experimental feature</p>
+  <p>OpenTelemetry Tracing is an experimental feature, meaning that the implementation or API might change at any point, with no compatibility guarantees. We'll do our best to make things work, but please use this feature at your own risk.</p>
+</div>
+
+To use OpenTelemetry Tracing, you'll need to enable the `opentelemetry-tracing` experiment, as well as specifying `--tracing-backend opentelemetry`.
```

Comment:
Will everyone reading this know how to enable the `opentelemetry-tracing` experiment? It might be helpful to list both flags that are required together: `--tracing-backend opentelemetry --experiment opentelemetry-tracing`.

Reply:
@yob thanks for the insight! Right into an update to this PR it'll go.